e2eTest/build.gradle

```diff
@@ -43,15 +46,19 @@ android {
     productFlavors {
         local {
             dimension "backend"
+            buildConfigField "boolean", "USE_EMULATORS", "true"
         }
         dev {
             dimension "backend"
+            buildConfigField "boolean", "USE_EMULATORS", "false"
```

Comment:
Can we remove other flavors here, since they're not intended to be used?

Reply:
I tried this out when I was constructing this file... the weird thing is that the build fails when these other flavors are not defined. Same as in `/sharedTest`